### PR TITLE
[Review Requested] 10mil Records in Secondary Database

### LINKED ISSUE #3: write data generation script - 10+ million records for secondary database

### PR DESCRIPTION
**Pull request by MengRS**
@jondionowens (primary)
@cjm771 (secondary)

fixes #3, #5, #6

Expected Behavior:

Run `npm run seed-large-database`.

Expected Behavior:
- 10 million records should generate in the `rooms` table of the `rooms` db in postgresql.  
- Any query should run under 50ms.
- Seed script should take no longer than 15 minutes.
- After running `npm start`, going to [http://localhost:3001/details/1](http://localhost:3001/details/1) should yield a data shape similar to the following:

### CODE REVIEW

**Comment by MengRS** [comment]
@jondionowens, please repull when you have a chance - I fixed some merge issues.

**Review by allenhsieh — CHANGES_REQUESTED** [review]
Hey so quick note right out the gate, we're talking about this on slack but the initial seed command doesn't work because it's looking for a folder that doesn't exist

File: package.json [hunk 1]
```diff
@@ -9,9 +9,11 @@
...
     "react-dev": "webpack -d --watch",
+    "start-dev": "node server/server.js",
     "server-dev": "NODE_ENV=development nodemon server/server.js",
     "seed-database": "node ./database/seed.js",
-    "seed-large-database": "node ./database/seedInsert.js",
+    "seed-large-database-mongo": "node ./database/seedInsert.js",
+    "seed-large-database": "node ./database/seedPostGres.js",
     "test-jest": "jest",
     "build": "webpack -p",
     "start": "node server/server.js"
```
Comment (allenhsieh) [inline package.json:19]: Hey Meng! This still start script refers to mongo code

**Review by allenhsieh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (allenhsieh) [reply package.json:19]: image

**Review by allenhsieh — APPROVED** [review]
Approving + squash and merging.

Review of things we learned talked about:
1) `.gitkeep` is needed for git to track empty folders
2) the way the start script is right now, mongodb needed to be installed and running even though you're using postgres, consider separating it out or deleting

Just small comments, otherwise all good:
image

**Pull request merged**